src/main/java/org/broadinstitute/hellbender/tools/spark/bwa/BwaSpark.java

```diff
@@ -29,6 +33,9 @@
             fullName = "bwamemIndexImage")
     private String indexImageFile;
 
+    @Argument(doc = "do pairwise alignment", fullName = "pairedMode", optional = true)
```

Comment:
This should be inverted to do single ended alignment.  We typically make flags default to false since it's a bit weird to have to specify a value for the flag.   It's fine to keep the option in all the other code as `pairedMode`, just invert this argument.